web/test/pages/profile/profile-helpers.tsx

```diff
@@ -161,3 +164,20 @@ export const expectLocationNotSavedAndError = (): void => {
   ).toBeInTheDocument();
   expect(screen.getByTestId("snackbar-alert-ERROR")).toBeInTheDocument();
 };
+
+export const getProfileDataMixin = (businessPersona: BusinessPersona): Partial<ProfileData> => {
+  return businessPersona === "FOREIGN"
+    ? {
+        foreignBusinessType: "NEXUS",
+        foreignBusinessTypeIds: ["NEXUS"],
+        nexusLocationInNewJersey: true,
+      }
+    : {};
+};
+export const phasesWhereGoToProfileShow = OperatingPhases.filter(
+  (it) => it.displayProfileOpportunityAlert
+).map((it) => it.id);
+
+export const phasesWhereGoToProfileDoNotShow = OperatingPhases.filter(
```

Comment:
[pebble] This is not grammatically correct, should be DoesNotShow

Reply:
resolved